scripts/install-sysdig.in

```diff
@@ -196,3 +196,5 @@ elif [ -f /etc/system-release-cpe ]; then
 else
 	unsupported
 fi
+
+modprobe -r _COMPONENT__probe
```

Comment:
Your description says you're loading a module.  `man modprobe` says:

> -r, --remove   
>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;This option causes modprobe to remove rather than insert a module.

Reply:
this has been changed to remove -r flag in latest commit

Reply:
reverting change; the reason why we have the -r flag is because sysdig automatically loads the newest probe, but in a situation where we are upgrading from a previous sysdig version the newer probe cannot be loaded because the older ver. probe is already loaded and has the same name as the newer probe module.  This change ensures this won't happen.

Reply:
does the probe get unloaded when you exit an instance of sysdig?

Reply:
@speedyguy17 no, it stays loaded.